sdk.go

```diff
@@ -1,27 +1,38 @@
 package sdk
 
+import "log"
+
 // RPCClient is a client to manage all rpc calls
 type RPCClient interface {
 	Call(result interface{}, method string, args ...interface{}) error
 }
 
-// SDK is a set of tools to interact with status node
-type SDK struct {
+// SDK is an interface to interact with status node
```

Comment:
afaics this interface is only used for testing, so, i wouldn't make it public. 

If its not public anymore you can keep SDK name for the basic struct, and rename this interface to sdk.

Additionally your interface (and mocks) are only needing to mock `Call` method, so, better not define all public methods here.

Have a look at this [section of Go CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#interfaces) it goes through this specific case :)